storybook/.storybook/storybook-elm/addon/register.js

```diff
@@ -0,0 +1,18 @@
+const React = require("react")
+const { addons } = require("@storybook/addons")
+const { StoryPanel } = require("./StoryPanel.js")
+
+const ADDON_ID = "ryannhg/elm-source"
+const PANEL_ID = `${ADDON_ID}/panel`
+
+addons.register(ADDON_ID, (api) => {
+    addons.addPanel(PANEL_ID, {
+        title: "Source",
+        paramKey: "elm-source",
+        render: ({ active, key }) => {
+            return (active)
+                ? React.createElement(StoryPanel, { className: "banana", key, api }, null)
```

Comment:
whats `"banana"` for?

Reply:
This comes from [here](https://github.com/ryannhg/elm-storybook-example/blob/aaaf4eecbe56496ca7f09675bea77fbaba1f825b/.storybook/storybook-elm/addon/register.js#L14)

Reply:
I'll have a closer look at what parts of the code are holdovers from the example